src/utils.py

```diff
@@ -99,3 +108,59 @@ def print_os_environment():
     """
     for key, val in os.environ.items():
         print(f"{key}: {val}")
+
+
+def set_commit_status_success_semver(commit):
+    commit.create_status(
+        state="success",
+        description="Semver label exists",
+        context=BLOCK_MERGE_SEMVER_CONTEXT,
+    )
+
+
+def set_commit_status_pending_no_semver(commit):
+    commit.create_status(
+        state="pending",
+        description="Missing Semver label",
+        context=BLOCK_MERGE_SEMVER_CONTEXT,
+    )
+
+
+def check_body_for_semver_labels(body):
+    add_labels = []
+    remove_labels = []
+
+    match = re.match(SEMVER_LABELS_REGEX, body, re.IGNORECASE)
+    if match:
+        label = match.group("label").lower()
+        add_labels = SEMVER_LABELS_MAP[label][ADD_LABELS]
+        remove_labels = SEMVER_LABELS_MAP[label][REMOVE_LABELS]
+
+    return add_labels, remove_labels
+
+
+def semver_labels_exist_in_pr(pull, body):
+    add_semver_labels, remove_semver_labels = check_body_for_semver_labels(body=body)
+    return (add_semver_labels or remove_semver_labels) and add_remove_labels(
+        pull=pull, add_labels=add_semver_labels, remove_labels=remove_semver_labels
+    )
+
+
+def add_remove_labels(pull, add_labels, remove_labels):
+    new_labels = add_labels.copy()
+    found_existing_label = False
+
+    current_labels = get_labels(pull=pull)
+    for label in current_labels:
```

Comment:
then here you can call `get_semver_label` and add, remove and set commit_status here.

Reply:
Please rename `label` to `label_from_pull`